docs/known_issues.rst

```diff
@@ -388,3 +388,37 @@ this can be fixed by updating to the latest version of the ``_astropy_init.py``
 file. The root cause of this issue is that pytest now tries to pick up the
 top-level ``test()`` function as a test, so we need to make sure that we set a
 ``test.__test__`` attribute on the function to ``False``.
+
+
+Workarounds for Unspported Versions
+===================================
+
+These are problems in *earlier* versions of Astropy which are caused by
+changes to something outside of the control of Astropy itself.  Workarounds
+are provided as a way to potentially keep code that depends on older versions
+functioning.  This are to be considered "as-is" or "best effort", though, as
```

Comment:
```suggestion
of older versions.  These are to be considered "as is" or "best effort," though, as
```

Reply:
I think these two lines were meant as "keep (code that depends on older versions of astropy) functioning", so it should be rephrased to "support code that depends on older versions of astropy"  (using "support" as a replacement for "keep ... functioning")